android/src/test/java/com/amplitude/android/AmplitudeRobolectricTests.kt

```diff
@@ -0,0 +1,51 @@
+package com.amplitude.android
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.amplitude.core.events.BaseEvent
+import io.mockk.spyk
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class AmplitudeRobolectricTests {
+    private lateinit var amplitude: Amplitude
+
+    @ExperimentalCoroutinesApi
+    @Before
+    fun setup() {
+        val context = ApplicationProvider.getApplicationContext<Context>()
+
+        val apiKey = "test-api-key"
+        amplitude = Amplitude(apiKey, context) {
+            this.minTimeBetweenSessionsMillis = 100
+            this.optOut = true
```

Comment:
nit: to make the test suite more reusable I think we should set `optOut` in the `optOpt` specific test below and only set more common testing config here.